### DISCUSSION TITLE
Is there any way to edit or add new subscription to app store connect

### DISCUSSION THREAD

**Discussion by nurhazbiy** [imdhemy/laravel-in-app-purchases#249] [Q&A]
Hello, i can't find it anywhere in documentation about how to make or edit subscription, whether it's auto renew or manual subscription. i found how to make a product, but it's only from app store connect console itself, not using this package. i now it's possible because there are api available from apple documentation, and it should be possible for this package to communicate with app store connect (because thats how we do server validation IAP on app store), but is it implemented yet? or do we need to implement it ourselves?

**Comment by imdhemy** [comment]
LIAP is not meant to support this feature, but it's possible to be added in the future. 

Could you please open a new issue for this feature request and add links to the relevant documentation?
